tabular/src/autogluon/tabular/task/tabular_prediction/predictor.py

```diff
@@ -816,6 +817,120 @@ def fit_weighted_ensemble(self, base_models: list = None, name_suffix='_custom',
+            In this case, not all of the original training data rows will have an OOF prediction. It is recommended to either drop these rows during the join or to get direct predictions on the missing rows via `predictor.predict`.
+
+        Parameters
+        ----------
+        model : str (optional)
+            The name of the model to get out-of-fold predictions from. Defaults to None, which uses the highest scoring model on the validation set.
+            Valid models are listed in this `predictor` by calling `predictor.get_model_names()`
+        transformed : bool, default = False
+            Whether the output values should be of the external label representation (False) or the internal label representation (True).
+            Generally, most users will want the external representation and keep `transformed=False`.
+        as_multiclass : bool, default = False
+            Whether to return binary classification probabilities as if they were for multiclass classification.
+                Output will contain two columns, and if `transformed=False`, the column names will correspond to the binary class labels.
+                The columns will be the same order as `predictor.class_labels`.
+            Only impacts output for binary classification problems.
+
+        Returns
+        -------
+        :class:`pd.Series` or :class:`pd.DataFrame` object of the out-of-fold training prediction probabilities of the model.
+        """
+        if not self._trainer.bagged_mode:
+            raise AssertionError('Predictor must be in bagged mode to get out-of-fold predictions.')
+        if model is None:
+            model = self.get_model_best()
+        y_pred_proba_oof_transformed = self.transform_features(base_models=[model], return_original_features=False)
+        if self.problem_type == MULTICLASS:
+            y_pred_proba_oof_transformed.columns = copy.deepcopy(self._learner.label_cleaner.ordered_class_labels_transformed)
+        else:
+            y_pred_proba_oof_transformed.columns = [self.label_column]
+            y_pred_proba_oof_transformed = y_pred_proba_oof_transformed[self.label_column]
+            if as_multiclass and self.problem_type == BINARY:
+                y_pred_proba_oof_transformed = LabelCleanerMulticlassToBinary.convert_binary_proba_to_multiclass_proba(y_pred_proba_oof_transformed, as_pandas=True)
+                if not transformed:
+                    y_pred_proba_oof_transformed.columns = copy.deepcopy(self._learner.label_cleaner.ordered_class_labels)
+        if transformed:
+            return y_pred_proba_oof_transformed
+        else:
+            return self.transform_labels(labels=y_pred_proba_oof_transformed, inverse=True, proba=True)
+
+    # TODO: v0.1 Should this instead return None when not valid instead of raising exception? Should this be a property or variable instead?
+    # TODO: v0.1 Add positive_class parameter to task.fit
+    def positive_class(self):
```

Comment:
Tutorials should be updated to reflect this new way of getting positive class.

Reply:
Good catch, added. Let me know if it needs to be changed in any other tutorials.

Reply:
Still missing update in the quick-start tutorial as well. Search for the cell:
```
pred_probs = predictor.predict_proba(test_data_nolab)
positive_class = [label for label in predictor.class_labels if predictor.class_labels_internal_map[label]==1][0]  # which label is considered 'positive' class
print(f"Predicted probabilities of class '{positive_class}':", pred_probs)
```

Reply:
added